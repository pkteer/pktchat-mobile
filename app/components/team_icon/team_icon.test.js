// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import {shallow} from 'enzyme';
import React from 'react';

import Preferences from '@mm-redux/constants/preferences';

import TeamIcon from './team_icon';

describe('TeamIcon', () => {
    const baseProps = {
        testID: 'team_icon',
        displayName: 'display-name',
        lastIconUpdate: 1,
        teamId: 'team-id',
        styleContainer: {},
        styleText: {},
        styleImage: {},
        theme: Preferences.THEMES.denim,
    };

    test('should match snapshot', () => {
        const wrapper = shallow(<TeamIcon {...baseProps}/>);

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
