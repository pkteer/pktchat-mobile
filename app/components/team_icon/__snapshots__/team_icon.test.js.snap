// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamIcon should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#1e325c",
        "borderRadius": 2,
        "height": 30,
        "justifyContent": "center",
        "width": 30,
      },
      Object {},
    ]
  }
  testID="team_icon"
>
  <FastImage
    onError={[Function]}
    source={
      Object {
        "uri": "/api/v4/teams/team-id/image?_=1",
      }
    }
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "borderRadius": 2,
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {},
      ]
    }
    testID="team_icon.content"
  />
</View>
`;
